web/packages/teleterm/src/services/config/configService.ts

```diff
@@ -85,12 +85,7 @@ const createAppConfigSchema = (platform: Platform) => {
     'keymap.openQuickInput': omitStoredConfigValue(
       z.string().default(defaultKeymap['open-quick-input'])
     ),
-    'fonts.sansSerifFamily': omitStoredConfigValue(
-      z.string().default(defaultFonts['sansSerif'])
-    ),
-    'fonts.monoFamily': omitStoredConfigValue(
-      z.string().default(defaultFonts['mono'])
-    ),
+    'fonts.monoFamily': z.string().default(defaultFonts['mono']),
```

Comment:
Since we want to set only the terminal font and possibly want the size to be adjustable as well, perhaps we should follow VSCode here and have settings such as `terminal.fontFamily` and `terminal.fontSize`.

Reply:
I like `terminal.fontSize` idea!
I will rename the properties too, it makes sense to keep them in a single namespace.